git-coco

```diff
@@ -291,6 +291,10 @@ prlist() {
         author="$(echo $author | awk -F '/' '{print $NF}')"
         dst_branch="$(echo "${dst_branch/refs\/heads\/}")"
         src_branch="$(echo "${src_branch/refs\/heads\/}")"
+        
+        if [ "${#title}" -gt 60 ]; then
+            title="$(echo "${title:0:57}")..."
```

Comment:
The "`$(echo)`" is redundant here because we can just directly capture the value after trimming

Reply:
thanks, fixed